webrtc-stats.html

```diff
@@ -986,9 +986,11 @@ <h2>
                 <p>
                   Total number of RTP packets received for this <a>SSRC</a>. At the receiving endpoint,
                   this is calculated as defined in [[!RFC3550]] section 6.4.1. At the sending
-                  endpoint the {{packetsReceived}} can be calculated by subtracting the packets lost
-                  from the expected Highest Sequence Number reported in the <a>RTCP Sender Report</a> as
-                  discussed in Appendix A.3. in [[!RFC3550]].
+                  endpoint the {{packetsReceived}} is estimated by subtracting the Cumulative Number of Packets Lost
+                  from the Extended Highest Sequence Number Received, both reported in the <a>RTCP Receiver
+                  Report</a>, and then subtracting the initial Extended Sequence Number that was sent to this SSRC in a
```

Comment:
If we start at seq num 10 and we get an RR saying that the extended highest seq num received was 10 (i.e. we got that packet!) and packets lost is 0, we would expct packetsReceived to be 1, right?

To me it sounds like this PR is off by one:
last seq num received - packets lost - initial seq num =
10 - 0 - 10 = 0

Does this need to say + 1?